Plate_2_data/4.processing_features/scripts/4.feature_select_sc_plate2.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Perform feature selection on normalized data
+
+# ## Import libraries
+
+# In[1]:
+
+
+import sys
+import pathlib
+
+import pandas as pd
+
+from pycytominer import feature_select
+from pycytominer.cyto_utils import output
+
+
+# ## Set paths and variables
+
+# In[2]:
+
+
+# directory where parquet files are located
+data_dir = pathlib.Path("./data/")
+
+# define input path
+normalized_file_path = str(pathlib.Path(f"{data_dir}/SHSY5Y_sc_norm.parquet"))
+
+# define ouput path
+feature_select_output_file = str(pathlib.Path(f"{data_dir}/SHSY5Y_sc_norm_fs.parquet"))
+
+
+# ## Perform feature selection
+
+# In[3]:
+
+
+# list of operations for feature select function to use on input profile
+feature_select_ops = [
+    "variance_threshold",
+    "correlation_threshold",
+    "blocklist",
+]
+
+# process each run
+# for SHSY5Y_run, info in run_info_dictionary.items():
+normalized_df = pd.read_parquet(normalized_file_path)
+
+print(f"Performing feature selection on normalized annotated merged single cells!")
+
+# perform feature selection with the operations specified
+feature_select_df = feature_select(
+    normalized_df,
+    operation=feature_select_ops,
+    output_file="none",
```

Comment:
Consider removing this specification (the default should be "none").

Reply:
👍 Missed that! Now removed.